src/helpers/parameters/checkDuplication.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
where are you using this check

Reply:
In route: `line 20`
https://github.com/andela/ninjas-ah-backend/blob/develop/src/routes/api/articles.js